src/main/java/org/testng/TimeBombSkipException.java

```diff
@@ -22,10 +22,11 @@
 
   private static final long serialVersionUID = -8599821478834048537L;
 
-  private static final SimpleDateFormat SDF= new SimpleDateFormat("yyyy/MM/dd");
+  private static final String FORMAT = "yyyy/MM/dd";
+  private final SimpleDateFormat sdf= new SimpleDateFormat(FORMAT);
   private Calendar m_expireDate;
-  private DateFormat m_inFormat= SDF;
-  private DateFormat m_outFormat= SDF;
+  private DateFormat m_inFormat= sdf;
```

Comment:
Is it possible to make them `final`?

Reply:
No its not possible to make these data members as `final` because they are being initialized by a few other constructors as well. That was why I chose to leave them as is and not mark them as `final`

Reply:
ok